src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  transition: color 0.3s, transform 0.3s;
+  cursor: pointer;
+
+  &:hover {
+    color: #34568b;
```

Comment:
you should change the color of the title directly, because, if you set the color of your title, this hover effect will not work